src/components/AcyPoolComponent/index.js

```diff
@@ -1,104 +1,482 @@
+import { useWeb3React } from '@web3-react/core';
 import { useConstantLoader } from '@/constants';
-import { GlpSwapBox, GlpSwapDetailBox } from '../PerpetualComponent/components/GlpSwapBox';
-import Reader from '@/acy-dex-futures/abis/Reader.json'
-import GlpManager from '@/acy-dex-futures/abis/GlpManager.json'
-import Glp from '@/acy-dex-futures/abis/ERC20.json'
-import useSWR from 'swr'
+import { getTokens, getContract } from '@/constants/future_option_power.js';
+import { useConnectWallet } from '@/components/ConnectWallet';
+import { approveTokens, addLiquidity, removeLiquidity } from '@/services/derivatives';
 import {
-  USD_DECIMALS,
-  GLP_DECIMALS,
-  PLACEHOLDER_ACCOUNT,
-  expandDecimals,
-  fetcher,
-  getInfoTokens,
   bigNumberify,
-} from '@/acy-dex-futures/utils'
-import { useChainId } from '@/utils/helpers';
-import { getTokens, getContract } from '@/constants/powers.js';
+  parseValue,
+  fetcher,
+  expandDecimals,
+  formatAmount,
+  formatAmountFree,
+  PLACEHOLDER_ACCOUNT,
+  ALP_DECIMALS,
+  USD_DECIMALS,
+  LONGDIGIT,
+  SHORTDIGIT,
+  BASIS_POINTS_DIVISOR,
+  MINT_BURN_FEE_BASIS_POINTS,
+  BURN_FEE_BASIS_POINTS,
+} from '@/acy-dex-futures/utils';
+import PerpetualTabs from '@/components/PerpetualComponent/components/PerpetualTabs';
+import BuyInputSection from '@/pages/BuyGlp/components/BuyInputSection';
+import glp40Icon from '@/pages/BuyGlp/components/ic_glp_40.svg'
+import AcyPerpetualButton from '@/components/PerpetualComponent/components/AcyPerpetualButton';
+import ERC20 from '@/acy-dex-futures/abis/ERC20.json';
+import Token from '@/acy-dex-futures/abis/Token.json';
+import Router from '@/abis/future-option-power/Router.json';
+import Reader from '@/abis/future-option-power/Reader.json';
+import Alp from '@/abis/future-option-power/Alp.json'
 
 import styles from './styles.less';
 
 const AcyPoolComponent = props => {
 
-  const { account, library, perpetuals } = useConstantLoader();
-  let chainId = 80001;
-
-  const tokens = getTokens(chainId);
-  const whitelistedTokens = tokens.filter(t => t.symbol !== "USDG")
-  const tokenAddresses = tokens.map(token => token.address)
-  // const readerAddress = perpetuals.getContract("Reader")
-  // const vaultAddress = perpetuals.getContract("Vault")
-  // const nativeTokenAddress = perpetuals.getContract("NATIVE_TOKEN")
-  // const glpManagerAddress = perpetuals.getContract("GlpManager")
-  // const glpAddress = perpetuals.getContract("GLP")
-  const readerAddress = getContract(chainId, "Reader");
-  const vaultAddress = getContract(chainId, "Vault");
-  const usdgAddress = getContract(chainId, "USDG");
-  const nativeTokenAddress = getContract(chainId, "NATIVE_TOKEN");
-  const routerAddress = getContract(chainId, "Router");
-  const glpManagerAddress = getContract(chainId, "GlpManager");
-  const glpAddress = getContract(chainId, "GLP");
-  const rewardRouterAddress = getContract(chainId, "RewardRouter");
+  const { account } = useConstantLoader(props)
```

Comment:
We don't use `useContantLoader()`, and `account` can be retrieved from `useWeb3React()`, as in line 43 below.